front/app/containers/MainHeader/index.tsx

```diff
@@ -9,7 +9,7 @@ import NotificationMenu from './NotificationMenu';
 import DesktopNavbar from './DesktopNavbar';
 import UserMenu from './UserMenu';
 import TenantLogo from './TenantLogo';
-import LoadableLanguageSelector from 'components/Loadable/LanguageSelector';
+import LanguageSelector from 'containers/MainHeader/LanguageSelector';
```

Comment:
This component seems super simple and was previously loading when the user moused over the container... since it's an important part of the UI and a small component it seems better to just load it directly. Let me know if I'm missing something...